include/aos/common/tools/log.hpp

```diff
@@ -114,6 +181,32 @@ class Log : private NonCopyable {
      */
     static void SetCallback(LogCallback callback) { GetCallback() = callback; }
 
+    /**
+     * Specialized Value method for FieldEntry values
+     */
+    static FieldEntry Value(const FieldEntry& field) { return field; }
+
+    /**
+     * Variadic template for Values with FieldEntry and more arguments
+     *
+     * @tparam N number of arguments.
+     * @param fields array of FieldEntry.
+     * @return StaticArray<FieldEntry, cMaxKeyValuePairs>
+     */
+    template <int N>
+    static StaticArray<FieldEntry, cMaxKeyValuePairs> Values(const FieldEntry (&fields)[N])
```

Comment:
Values take too much stack. Probably we need to remove it and use only Value. Let's discuss with team.

Reply:
Removed